### DISCUSSION TITLE
Configuring SDDM

### DISCUSSION THREAD

**Discussion by dudenukm** [prasanthrangan/hyprdots#483] [Q&A] [answered]
How do I position the login bar in the middle?

**Comment by T-Crypt** [comment, marked as answer]
You need to edit the theme.conf file in 

/use/share/sddm/themes/corners/

**Comment by dudenukm** [comment]
I checked '/etc/sddm',
if i Change
'Current=<different directory>'
would it crash?

**Reply by dudenukm** [reply]
I was able to theme it. Thanks
